hazelcast-client/src/test/java/com/hazelcast/client/replicatedmap/nearcache/ClientReplicatedMapNearCacheBasicTest.java

```diff
@@ -119,4 +119,28 @@ public void whenCacheIsFull_thenPutOnSameKeyShouldUpdateValue_withUpdateOnNearCa
     @Ignore(value = "The ClientReplicatedMapProxy is missing `invalidateNearCache(keyData)` calls")
     public void whenPutAllIsUsed_thenNearCacheShouldBeInvalidated_withUpdateOnNearCacheAdapter() {
     }
+
+    @Test(expected = UnsupportedOperationException.class)
```

Comment:
The `ReplicatedMap` has no `replace()` methods, so we expect the `UnsupportedOperationException` from the `DataStructureAdapter` here.